_posts/2021/2021-09-20-compact_blank-remove-empty-strings-from-array-and-hash.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Use compact_blank to remove empty strings from Arrays and Hashes"
+description: "This is one of those “Nice Things” in Rails"
+layout: article
+category: ruby
+image:
+  base: '2021/compact_blank-remove-empty-strings-from-array-and-hash'
+  alt: "Compact Disc"
+  credit: "Jan Huber"
+  source: "https://unsplash.com/photos/oTCRizM-PUI"
+
+---
+
+Active Support is a library of “useful” things that supports (!) the other parts of Rails. It includes ways to [extend the base Ruby classes](https://guides.rubyonrails.org/active_support_core_extensions.html), such as `Hash`, `String` and `Integer`.
+
+Even the most experienced Rails developer is constantly finding helpful methods in Active Support that are new to them. And useful methods are continually being added in new versions of the library.
```

Comment:
*constantly discovering helpful methods... (don't need the 'that are new to them' part then, I think).

Reply:
```suggestion
Even the most experienced Rails developer is constantly finding helpful methods in Active Support. And useful methods are continually being added in new versions of the library.
```